### PR TITLE
Adding maintainers guidelines

### PR DESCRIPTION
**Pull request by jakubhajek**
### What does this PR do?

The PR adds guidelines for the maintainers and revamp a little bit of the contributing section.

### Motivation

We would like to engage new maintainers in our project. That's why we created some guidelines to make the process easier and help contributors to become maintainers of the project.

### More

- [x] Added/updated documentation

### Additional Notes

N/A

### CODE REVIEW

File: docs/content/contributing/maintainers-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# The Maintainers Guidelines
+
+Note: the document is a work in progress.
+
+Welcome to the Traefik Community. This document describes various responsibilities and guidelines for Traefik maintainers. We are in the process of hiring  maintainers into the project, so that information can be relevant if you are interested in being a part of the community.
```
Comment (emilevauge) [inline docs/content/contributing/maintainers-guidelines.md]:
```suggestion
Welcome to the Traefik Community. This document describes how to be part of the core team as well as various responsibilities and guidelines for Traefik maintainers.. We are strongly promoting a philosophy of openness and sharing, and firmly standing against the elitist closed approach. Being part of the core team should be accessible to anyone who is motivated and want to be part of that journey!
```

File: README.md [hunk 2]
```diff
@@ -124,6 +125,10 @@ git clone https://github.com/traefik/traefik
 
 You can find high level and deep dive videos on [videos.traefik.io](https://videos.traefik.io).
 
+## Maintainers Guidelines
+
+[The Maintainers Guidelines](docs/content/contributing/maintainers-guidelines.md)
```
Comment (emilevauge) [inline README.md]:
```suggestion
We are strongly promoting a philosophy of openness and sharing, and firmly standing against the elitist closed approach. Being part of the core team should be accessible to anyone who is motivated and want to be part of that journey!
This [document](docs/content/contributing/maintainers-guidelines.md) describes how to be part of the core team as well as various responsibilities and guidelines for Traefik maintainers.
You can also find more information on our process to review pull requests and manage issues [in this document](docs/content/contributing/maintainers.md).
```

File: README.md [hunk 3]
```diff
@@ -124,6 +125,10 @@ git clone https://github.com/traefik/traefik
 
 You can find high level and deep dive videos on [videos.traefik.io](https://videos.traefik.io).
 
+## Maintainers Guidelines
```
Comment (emilevauge) [inline README.md]: I would merge the two maintainers sections in one _Maintainers_ to simplify.

```suggestion
## Maintainers
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,4 +1,11 @@
...
 
-- https://doc.traefik.io/traefik/contributing/submitting-pull-requests/
-- https://doc.traefik.io/traefik/contributing/submitting-issues/
+Here are some guidelines that should help to start contributing to the project.
+
+- [Submitting pull Requests](https://github.com/traefik/contributors-guide/blob/master/pr_guidelines.md)
+- [Submitting issues](https://doc.traefik.io/traefik/contributing/submitting-issues/)
+- [Submitting security issues](docs/content/contributing/submitting-security-issues.md)
+
+Should you have a look at [external maintainers guidelines](docs/content/contributing/external-maintainers-guidelines.md)?
```
Comment (emilevauge) [inline CONTRIBUTING.md]: We should remove the _external_ mention.

```suggestion
Should you have a look at [maintainers guidelines](docs/content/contributing/external-maintainers-guidelines.md)?
```

File: docs/content/contributing/maintainers-guidelines.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+- Documentation contribution. 
+  - Technical documentation is one of the most important components of the product. The ability to set up a testing environment using Traefik in a few minutes using the official documentation is a game changer.
+- Documentation contribution. 
+  - Technical documentation is one of the most important components of the product. The ability to set up a testing environment using Traefik in a few minutes using the official documentation is a game changer.
+- You will be listed on our Maintainers Github page as well as on our website in the section [maintainers](maintainers.md).
+- We will be promoting you in social channels (mostly on Twitter).
+
+## Governance
+
+- Roadmap meetings on a regular basisc where all maintainers are welcome.
```
Comment (emilevauge) [inline docs/content/contributing/maintainers-guidelines.md]:
```suggestion
- Roadmap meetings on a regular basis where all maintainers are welcome.
```

File: docs/content/contributing/maintainers-guidelines.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+- We publish a report to the channel on a daily basis that includes all the activities performed during the day. You are updated in regards to the workload that has been processed including: working on the new features and enhancements, activities related to the reported issues and PR’s, other important project-related announcements.
+- At 5:00 PM CET every day we review all the created issues that have been reported, assign the specific *[labels](maintainers.md#Labels)* and prioritize them based on the severity of the problem. The process is called *[issue triaging](https://github.com/traefik/contributors-guide/blob/master/issue_triage.md).* Each of the maintainers is welcome to join the meeting. For that purpose, we use a dedicated Discord server where you are invited once you have become the official maintainer.
+
+## Maintainers Activity
+Every six months we will validate maintainers acttivity and determine if they are still active and participating as a maintainer.
```
Comment (emilevauge) [inline docs/content/contributing/maintainers-guidelines.md]:
```suggestion
In order to keep the core team efficient and dynamic, maintainers activity and involvement will be reviewed on a regular basis.
```

**Review by emilevauge — CHANGES_REQUESTED** [review]
Hey @jakubhajek, 👏 for this achievement.
A few comments here and there.

File: README.md [hunk 7]
```diff
@@ -28,6 +28,7 @@ Pointing Traefik at your orchestrator should be the _only_ configuration step yo
 . **[Support](#support)** .
 **[Release cycle](#release-cycle)** .
 **[Contributing](#contributing)** .
+**[Maintainers Guidelines](#maintainers-guidelines)** .
```
Comment (emilevauge) [inline README.md]: I guess it needs to be removed too.

File: README.md [hunk 8]
```diff
@@ -126,6 +127,12 @@ You can find high level and deep dive videos on [videos.traefik.io](https://vide
 
 ## Maintainers
 
+We are strongly promoting a philosophy of openness and sharing, and firmly standing against the elitist closed approach. Being part of the core team should be accessible to anyone who is motivated and want to be part of that journey!
+This [document](docs/content/contributing/maintainers-guidelines.md) describes how to be part of the core team as well as various responsibilities and guidelines for Traefik maintainers.
+You can also find more information on our process to review pull requests and manage issues [in this document](docs/content/contributing/maintainers.md).
+
+## Maintainers
```
Comment (emilevauge) [inline README.md]: Not sure why this section is still here 🤔

File: README.md (hunk 7 above)
Comment (jakubhajek) [reply README.md]: Correct, this is already removed.

File: README.md (hunk 8 above)
Comment (jakubhajek) [reply README.md]: I removed the duplicated header Maintainers. Not sure if we should remove the link to the Maintainers guidelines and to The Maintainers. I left it unchanged for now.

File: docs/content/contributing/maintainers-guidelines.md [hunk 9]
```diff
@@ -0,0 +1,70 @@
+# The Maintainers Guidelines
+
+Note: the document is a work in progress.
+
+Welcome to the Traefik Community. This document describes how to be part of the core team as well as various responsibilities and guidelines for Traefik maintainers.. We are strongly promoting a philosophy of openness and sharing, and firmly standing against the elitist closed approach. Being part of the core team should be accessible to anyone who is motivated and want to be part of that journey!
```
Comment (rtribotte) [inline docs/content/contributing/maintainers-guidelines.md]:
```suggestion
Welcome to the Traefik Community. This document describes how to be part of the core team as well as various responsibilities and guidelines for Traefik maintainers. We are strongly promoting a philosophy of openness and sharing, and firmly standing against the elitist closed approach. Being part of the core team should be accessible to anyone who is motivated and want to be part of that journey!
```

File: docs/content/contributing/maintainers-guidelines.md [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+- Participating in a daily [issue triage](https://github.com/traefik/contributors-guide/blob/master/issue_triage.md). 
+  - The process helps to understand and prioritize the reported issue according to its importance and severity. This is crucial in order to learn how our users implement Traefik. Each of the issues that are labeled as bug/possible bug/confirmed requires a reproducible use case. You can help in creating a reproducible use case if it has not been added to the issue or use the sample code provided by the reporter. Typically, a simple docker compose should be enough to reproduce the issue.
+- Code contribution. 
+- Documentation contribution. 
+  - Technical documentation is one of the most important components of the product. The ability to set up a testing environment using Traefik in a few minutes using the official documentation is a game changer.
+- Documentation contribution. 
+  - Technical documentation is one of the most important components of the product. The ability to set up a testing environment using Traefik in a few minutes using the official documentation is a game changer.
```
Comment (rtribotte) [inline docs/content/contributing/maintainers-guidelines.md]:
```suggestion
```

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -1,4 +1,10 @@
...
 
-- https://doc.traefik.io/traefik/contributing/submitting-pull-requests/
-- https://doc.traefik.io/traefik/contributing/submitting-issues/
+Here are some guidelines that should help to start contributing to the project.
+
+- [Submitting pull Requests](https://github.com/traefik/contributors-guide/blob/master/pr_guidelines.md)
+- [Submitting issues](https://doc.traefik.io/traefik/contributing/submitting-issues/)
+- [Submitting security issues](docs/content/contributing/submitting-security-issues.md)
+
+Should you have a look at [maintainers guidelines](docs/content/contributing/maintainers-guidelines.md)?
```
Comment (rtribotte) [inline CONTRIBUTING.md]:
```suggestion
If you are willing to become a maintainer of the project, please take a look at the [maintainers guidelines](docs/content/contributing/maintainers-guidelines.md).
```

**Review by rtribotte — COMMENTED** [review]
Here a few comments :-)
For the one on the reference to the maintainer's guidelines in `CONTRIBUTING.MD`, I was not sure of the intention, so never mind if it was turned into a question on purpose.

File: docs/content/contributing/maintainers-guidelines.md [hunk 12]
```diff
@@ -0,0 +1,68 @@
...
+    - Substantial review of at least one or two PRs from either contributors or maintainers.
+    - Opened at least one or two bug fixes or feature request PRs that were eventually merged (or on a trajectory for merge).
+    - Substantial participation in the Help Wanted program (answered questions, helped identify issues, applied guidelines from the Help Wanted guide to open issues).
+    - Substantial participation with the community in general.
+- Has the maintainer shown a consistent pattern of helpful, non-threatening, and friendly behavior towards other people on the maintainer team and with our community?
+
+## Additional comments for (not only) maintainers
+
+- Use Traefik on a daily basis. You can’t be a good maintainer if you are not using Traefik daily and you can’t put yourself in your users’ shoes.
+- Prioritize the maintainer. Be user-focused but if you use Traefik every day, you are already in the user's shoes, so respect other maintainers and other community members.
```
Comment (jbdoumenjou) [inline docs/content/contributing/maintainers-guidelines.md]: I'm not sure about the daily basis usage, perhaps we can rephrase with something like.
```suggestion
- Be able to put yourself in users’ shoes.
- Be open-minded and respectful with other maintainers and other community members.
```

File: docs/content/contributing/maintainers-guidelines.md [hunk 13]
```diff
@@ -0,0 +1,68 @@
...
+- Keep the communication public - if anyone tries to communicate with you directly, ask him politely to move the conversation to a public communication channel.
+- Stay away from defensive comments.
+- Please try to express your thoughts clearly enough and note that some of us (including me) are not native English speakers. Try to rephrase your sentences, avoiding mental shortcuts; none of us is able to predict your thoughts.
```
Comment (jbdoumenjou) [inline docs/content/contributing/maintainers-guidelines.md]:
```suggestion
- Please try to express your thoughts clearly enough and note that some of us are not native English speakers. Try to rephrase your sentences, avoiding mental shortcuts; none of us is able to predict your thoughts.
```

File: docs/content/contributing/maintainers-guidelines.md [hunk 14]
```diff
@@ -0,0 +1,68 @@
...
+
+Note: you do not have to meet all of the listed requirements, but must have achieved several.
+
+- Enabled [2FA](https://docs.github.com/en/github/authenticating-to-github/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication) on your Github account
+- The contributor has opened and successfully run medium to large PR’s in the past 6 months.
+- The contributor has participated in multiple code reviews of other PR’s, including those of other maintainers and contributors.
+- The contributor showed a consistent pattern of helpful, non-threatening, and friendly behaviour towards other community members in the past.
+- The contributor is active on Traefik Community forums or other technical forums / boards such as K8S slack, Reddit, StackOverflow, hacker news.
+- Have read and accepted the contributor guidelines.
+
```
Comment (jbdoumenjou) [inline docs/content/contributing/maintainers-guidelines.md:25]:
```suggestion
```

File: docs/content/contributing/maintainers-guidelines.md [hunk 15]
```diff
@@ -0,0 +1,68 @@
...
+- Documentation contribution. 
+  - Technical documentation is one of the most important components of the product. The ability to set up a testing environment using Traefik in a few minutes using the official documentation is a game changer.
+- You will be listed on our Maintainers Github page as well as on our website in the section [maintainers](maintainers.md).
+- We will be promoting you in social channels (mostly on Twitter).
+
+## Governance
+
+- Roadmap meetings on a regular basis where all maintainers are welcome.
+
+## Communicating
```
Comment (jbdoumenjou) [inline docs/content/contributing/maintainers-guidelines.md:44]:
```suggestion
## Communicating

```

File: docs/content/contributing/maintainers-guidelines.md [hunk 16]
```diff
@@ -0,0 +1,68 @@
...
+- We publish a report to the channel on a daily basis that includes all the activities performed during the day. You are updated in regards to the workload that has been processed including: working on the new features and enhancements, activities related to the reported issues and PR’s, other important project-related announcements.
+- At 5:00 PM CET every day we review all the created issues that have been reported, assign the specific *[labels](maintainers.md#Labels)* and prioritize them based on the severity of the problem. The process is called *[issue triaging](https://github.com/traefik/contributors-guide/blob/master/issue_triage.md).* Each of the maintainers is welcome to join the meeting. For that purpose, we use a dedicated Discord server where you are invited once you have become the official maintainer.
+
+## Maintainers Activity
```
Comment (jbdoumenjou) [inline docs/content/contributing/maintainers-guidelines.md:50]:
```suggestion
## Maintainers Activity

```

**Pull request closed without merging**